kernel/kernel-config-server/README.md

```diff
@@ -86,11 +103,7 @@ server.servlet.path=/config
 #Path inside the GIT repo where config files are stored, in our case they are inside config directory
 #spring.cloud.config.server.git.search-paths=<folder-in-git-repository-containing-configuration>
 
-# Uncomment spring.profiles.active and spring.cloud.config.server.native.search-locations for     # connecting to local file system for configuration.
-#################################################################
-# spring.profiles.active=native
-
-# spring.cloud.config.server.native.search-locations=file:///<config-location-on-your-system>
+spring.profiles.active=composite
```

Comment:
should we do this change in the readme file, since we will still support the native part when requrired.

Reply:
we can support native through composite profile https://github.com/mosip/commons/pull/1553#discussion_r1700132578